app/frontend/src/components/designer/settings/FormMetadataSettings.vue

```diff
@@ -0,0 +1,110 @@
+<script setup>
+import { storeToRefs } from 'pinia';
+import { useI18n } from 'vue-i18n';
+import { onMounted, ref } from 'vue';
+import { useFormStore } from '~/store/form';
+
+const { t, locale } = useI18n({ useScope: 'global' });
+const { form, isRTL } = storeToRefs(useFormStore());
+const techdocsLink =
+  'https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Integrations/Form-Metadata/';
+
+const jsonStr = ref(null);
+
+onMounted(async () => {
+  jsonStr.value = JSON.stringify(form.value.formMetadata.metadata, null, 2);
+});
+
+function updateMetadata(v) {
+  try {
+    form.value.formMetadata.metadata = JSON.parse(v);
+  } catch {
+    return false;
+  }
+}
+
+function formatJSON(focus) {
+  if (!focus) {
+    try {
+      const t = JSON.parse(jsonStr.value);
+      jsonStr.value = JSON.stringify(t, null, 2);
+    } catch {
+      /* empty */
+    }
+  }
+}
+
+/* c8 ignore start */
+const metadataRules = ref([
+  (v) => {
+    try {
+      JSON.parse(v);
```

Comment:
Don't know why, but this is passing validation:
![image](https://github.com/user-attachments/assets/03f3833e-064f-4d9d-a291-00eccc60d383)

Reply:
(that is, until the form is saved, and then a generic error is shown)